br/br-checkpoint.md

```diff
@@ -0,0 +1,40 @@
+---
+title: Checkpoint Backup
+summary: Learn about the checkpoint backup feature, including its application scenarios, usage, and implementation details.
+---
+
+# Checkpoint Backup <span class="version-mark">New in v6.5.0</span>
+
+Snapshot backup might end in advance due to recoverable errors, such as disk exhaustion and node crash. Before TiDB v6.5.0, data that is backed up before the interruption would be invalidated after the error is addressed, and you need to start the backup from scratch. For large clusters, this results in considerable extra cost.
+
+In TiDB v6.5.0, Backup & Restore (BR) introduces checkpoint backup feature to allow continuing an interrupted backup. This feature is enabled by default. After this feature is enabled, most data of the interrupted backup is retained after an unexpected exit.
+
+## Application scenarios
+
+If your TiDB cluster is large and cannot afford to back up again after a failure, you can enable the checkpoint backup feature. After this feature is enabled, br command-line tool (hereinafter referred to as `br`) periodically records the shards that have been backed up. In this way, the next backup retry can use the backup progress close to the abnormal exit.
+
+## Usage limitations
+
+During the backup, `br` periodically updates the `gc-safepoint` of the backup snapshot in PD to avoid data being garbage collected. When `br` exits, the `gc-safepoint` cannot be updated in time. As a result, before the next backup retry, the data might have been garbage collected.
+
+To avoid this situation, `br` keeps the `gc-safepoint` for about one hour by default when `gcttl` is not specified. If you need to extend this time, you can set the `gcttl` parameter.
```

Comment:
```suggestion
To avoid this situation, `br` keeps the `gc-safepoint` for about one hour by default when `gcttl` is not specified. You can set the `gcttl` parameter to extend the retention period if needed .
```